docs/atlascli/command/atlas-clusters-create.txt

```diff
@@ -43,7 +43,7 @@ Arguments
    * - name
      - string
      - false
-     - Name of the cluster. The cluster name cannot be changed after the cluster is created. Cluster name can contain ASCII letters, numbers, and hyphens. You must specify the cluster name argument if you don't use the --file option.
```

Comment:
bad merge? this was added recently

Reply:
My bad. Updated. Thanks for pointing it out!